db/migrate/20200502130958_create_merchant_payouts.rb

```diff
@@ -0,0 +1,21 @@
+class CreateMerchantPayouts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :merchant_payouts do |t|
+      t.integer :payout_type, default: 0, null:false
+      t.integer :total_amount, null: false
+      t.integer :account_number, null: false
```

Comment:
Not sure we wanna keep our account numbers in our DB. Could this be a string instead? Right now we have two bank accounts we're paying out from.